samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
`SPDX-License-Identifier: LicenseRef-Nordic-5-Clause` ?

Reply:
The content of the file was wrong. Updated to correct one.